README.md

```diff
@@ -329,6 +329,17 @@ buffer config ( only work `producer_type` = "async" )
     when `retryable` is `true` that means kafka server surely not committed this messages, you can safely retry to send;
     and else means maybe, recommend to log to somewhere.
 
+* `wait_on_buffer_full`
```

Comment:
please also note that it could not be used in those phases which do not support yields, i.e. log phase.

Reply:
ok👌